XenModel/XenAPI-Extensions/SR.cs

```diff
@@ -829,7 +837,10 @@ public override string ToString()
 
         public bool MultipathCapable()
         {
-            return "true" == Get(sm_config, "multipathable");
+            SM relatedSm = GetSM;
+            bool isAnSmCapability = relatedSm != null && relatedSm.MultipathEnabled;
+            bool isInSmConfig = "true" == Get(sm_config, "multipathable");

```

Comment:
You can use `bool isInSmConfig = BoolKey(sm_config, "multipathable")` for this instead.